tests/stubs/config.js

```diff
@@ -17,13 +17,13 @@ module.exports = {
    |
    |
    */
-  'default': 'local',
+  'default': 'ftp',
```

Comment:
This shouldn't change

Reply:
Fail, forgot to remove this line after dev :)